lib/game.rb

```diff
@@ -40,6 +40,36 @@ def self.all
     @@all
   end
 
+  def self.percentage_home_wins
+    home_win_counter = 0
+    @@all.each do |game|
+      if game.home_goals > game.away_goals
+        home_win_counter += 1
+      end
+    end
+    (home_win_counter / @@all.length.to_f).round(2) # 3237 / 7441
```

Comment:
We'll eventually want to get rid of the commented out portions before we turn in. But nice logic here.